lib/Runtime/ByteCode/ByteCodeDumper.cpp

```diff
@@ -877,8 +893,6 @@ namespace Js
             case OpCode::LdEnvObj:
             case OpCode::LdLocalObjSlot:
             case OpCode::LdParamObjSlot:
-                Output::Print(_u(" R%d = [%d] "),data->Value, data->SlotIndex);
```

Comment:
Is this intentional? You want the `Ld*Slot` opcodes to fall-through to the same path as function-handling opcodes? That looks incorrect to me.

Reply:
Definitely not intentional, I must have accidentally deleted this when I thought focus was on another window... I removed most instances of this type of mistake but I guess I missed one.

---
In reply to: [204227269](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r204227269) [](ancestors = 204227269)